rules/java_rules.build_defs

```diff
@@ -25,9 +24,11 @@ def java_library(name:str, srcs:list=None, src_dir:str=None, resources:list=[],
       javac_flags (list): List of flags passed to javac.
       labels (list): Additional labels to apply to this rule.
     """
+    toolchain = toolchain if toolchain else CONFIG.JAVA_TOOLCHAIN
```

Comment:
you should just be able to make `CONFIG.JAVA_TOOLCHAIN` the default for the argument, I think.